playbooks/demo_suma.yml

```diff
@@ -55,13 +58,13 @@
 #   register: output
 # - debug: var=output
 
-  - name: Wait for 10 seconds
-    pause:
-      seconds: 10
+#  - name: Wait for 10 seconds
```

Comment:
Why we comment off delay for reboot. don't remember delay is not needed when I run this suma playbook?

Reply:
The delay here was to wait for the system to reboot. But the reboot task is commented out. So we are just waiting.